recipes/openblas/bld.bat

```diff
@@ -0,0 +1,11 @@
+
+:: Set $HOME to the current dir so msys runs here
+set HOME=%cd%
+
+:: Configure, build, test, and install using `make`.
+bash -lc "make FC=gfortran DYNAMIC_ARCH=1 BINARY=$ARCH NO_LAPACK=0 NO_AFFINITY=1 USE_THREAD=1 PREFIX=`echo $LIBRARY_PREFIX | tr '\\' '/'`"
```

Comment:
Need to be careful here because the max thread that openblas will use is determined at compile time.  `NUM_THREADS=128` will set the maximum thread count to 128.  Then it can be controlled on the user side by the environment variable `OPENBLAS_NUM_THREADS`